dataservice/api/participant/schemas.py

```diff
@@ -21,19 +20,29 @@ class Meta(BaseSchema.Meta):
     _links = ma.Hyperlinks({
         'self': ma.URLFor(Meta.resource_url, kf_id='<kf_id>'),
         'collection': ma.URLFor(Meta.collection_url),
-        'study': ma.URLFor('api.studies', kf_id='<study_id>'),
+        'study': ma.URLFor('api.studies', kf_id='<study_id>')
     })
 
-    def dump(self, p, *args, **kwargs):
+    def dump(self, obj, *args, **kwargs):
```

Comment:
I think there may be a more generic way to do this (and rendering of all of our relation links whether parent or child) on the base schema, but I think it would require some refactoring and I'd like to get this first version merged in soon since it's a blocker for consumers.